prog/dftb+/lib_io/hsdparser.F90

```diff
@@ -131,28 +131,17 @@ module dftbp_hsdparser
   !> list label
   character(len=*), parameter :: attrList = "l"
 
-
-  !> Length of a parsed line
+  !> Length of a parsed line (adjust lineFormat below, if you change this number)
   integer, parameter :: lc = 1024
 
+  !> Format string to read in one parse line (field length must correspond to lc above)
+  character(len=*), parameter :: lineFormat = "(A1024)"
```

Comment:
Strong case for using preprocessing, to set 1024 only once for its use in 3 places (lc, lineFormat and MAXRECL).

Reply:
Makes sense, done. But not for `MAXRECL`, which contains 1024 only by accident.